client/src/hooks/useInterval.js

```diff
@@ -0,0 +1,21 @@
+import {useEffect, useRef } from 'react';
+
+export function useInterval(callback, delay) {
+  const savedCallback = useRef();
+
+  // Remember the latest callback.
+  useEffect(() => {
+    savedCallback.current = callback;
+  }, [callback]);
+
+  // Set up the interval.
+  useEffect(() => {
+    function tick() {
+      savedCallback.current();
+    }
+    if (delay !== null) {
+      const id = setInterval(tick, delay);
+      return () => clearInterval(id);
+    }
+  }, [delay]);
+}
```

Comment:
Should have an empty line at the end of the file, same with all other file you worked on